topcoffea/data/btag_sf_correctionlib/README.md

```diff
@@ -8,10 +8,14 @@ cp /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/2016preVFP_U
 cp /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/2016postVFP_UL/btagging.json.gz 2016postVFP_UL_btagging.json.gz
 cp /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/2017_UL/btagging.json.gz 2017_UL_btagging.json.gz
 cp /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/2018_UL/btagging.json.gz 2018_UL_btagging.json.gz
+cp /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/2022_Summer22/btagging.json.gz 2022_btagging.json.gz 
```

Comment:
Super minor thing, but just to help organize the readme visually, could you add a space above this line (so there is a space between the R2 and R2 ones)?